website/src/components/Property.tsx

```diff
@@ -0,0 +1,203 @@
+import clsx from 'clsx';
```

Comment:
![badge](https://img.shields.io/badge/review-memo-blue.svg)
copied from modular-form, and rewritten a little bit to avoid using Solid.js
https://github.com/fabian-hiller/modular-forms/blob/69701fec32d0cbf233b42cc7e3080eee060c3bde/website/src/components/Property.tsx#L1-L221